docs/resources/cce_cluster.md

```diff
@@ -192,6 +192,9 @@ The following arguments are supported:
 * `enterprise_project_id` - (Optional, String, ForceNew) The enterprise project ID of the CCE cluster.
   Changing this parameter will create a new cluster resource.
 
+* `tags` - (Optional, Map, ForceNew) Specifies the tags of the CCE cluster, key/value pair format.
+  Changing this parameter will create a new cluster resource.
```

Comment:
better to add update tags support instead of forcenew here, seems it's already support tags update with cluster on API side.

Reply:
ignore this ^